frontend/src/routes/ApplicationForm/FormStructure.js

```diff
@@ -42,7 +42,7 @@ const FormStructure = ({
           {myApplication && (
             <StyledSpan bold>
               <Moment format="dddd Do MMMM , \k\l. HH:mm:ss">
-                {myApplication.time_sent}
+                {myApplication.created_at}
```

Comment:
This value doesn't exist the first time you submit, without reloading the page
![image](https://user-images.githubusercontent.com/16608915/63725270-2c752f80-c85a-11e9-9147-22d6e7f938aa.png)

Reply:
Fixed with rebase.